Documentation/building/crossgen.md

```diff
@@ -34,7 +34,7 @@ The original MSIL assemblies are no longer needed by the runtime.
 
 If you installed CoreCLR from a NuGet package, `System.Private.CoreLib.ni.dll` is included in the package.
```

Comment:
This should be fixed the other way. `System.Private.CoreLib.ni.dll` is not included in the package. It was renamed to `System.Private.CoreLib.dll`. We intentionally use same name regardless of whether the file is native image or plain IL Image.

Reply:
All native DLLs should have the same name as the IL version by default.